### ISSUE TITLE
Add digest authentication support.

### ISSUE THREAD

**Issue by jkjamies** [timestee/dart-webdav#12]
Would be nice if it supported both digest and basic authentication.

**Comment by timestee** [comment]
> Would be nice if it supported both digest and basic authentication.

good point, but should waiting for a fews days, or you could pull request

**Comment by timestee** [comment]
@jkjamies good point, but should waiting for a fews days, or you could pull request
